sdk/src/client/node_api/core/mod.rs

```diff
@@ -31,51 +31,45 @@ impl Client {
         futures::future::try_join_all(output_ids.iter().map(|id| self.get_output(id))).await
     }
 
-    /// Requests outputs by their output ID in parallel, ignoring failed requests.
+    /// Requests outputs by their output ID in parallel, ignoring outputs not found.
     /// Useful to get data about spent outputs, that might not be pruned yet.
-    pub async fn get_outputs_ignore_errors(&self, output_ids: &[OutputId]) -> Result<Vec<Output>> {
-        Ok(
-            futures::future::join_all(output_ids.iter().map(|id| self.get_output(id)))
-                .await
-                .into_iter()
-                .filter_map(Result::ok)
-                .collect(),
-        )
+    pub async fn get_outputs_ignore_not_found(&self, output_ids: &[OutputId]) -> Result<Vec<Output>> {
+        futures::future::join_all(output_ids.iter().map(|id| self.get_output(id)))
+            .await
+            .into_iter()
+            .filter(|res| !matches!(res, Err(Error::Node(NodeApiError::NotFound(_)))))
+            .collect()
     }
 
     /// Requests metadata for outputs by their output ID in parallel.
     pub async fn get_outputs_metadata(&self, output_ids: &[OutputId]) -> Result<Vec<OutputMetadata>> {
         futures::future::try_join_all(output_ids.iter().map(|id| self.get_output_metadata(id))).await
     }
 
-    /// Requests metadata for outputs by their output ID in parallel, ignoring failed requests.
-    pub async fn get_outputs_metadata_ignore_errors(&self, output_ids: &[OutputId]) -> Result<Vec<OutputMetadata>> {
-        Ok(
-            futures::future::join_all(output_ids.iter().map(|id| self.get_output_metadata(id)))
-                .await
-                .into_iter()
-                .filter_map(Result::ok)
-                .collect(),
-        )
+    /// Requests metadata for outputs by their output ID in parallel, ignoring outputs not found.
+    pub async fn get_outputs_metadata_ignore_not_found(&self, output_ids: &[OutputId]) -> Result<Vec<OutputMetadata>> {
+        futures::future::join_all(output_ids.iter().map(|id| self.get_output_metadata(id)))
```

Comment:
hmmm ... may still want to use `try_join_all` here? Otherwise the behavior is a bit different and not just ignoring outputs not found.

Reply:
No, `try_join_all` will stop at the first error

Reply:
yeah, that's fine but probably should be stated in the doc?

Reply:
Idk I think it's stated already `ignoring outputs not found`. What do you want to add ?